content/en/seameet/seameet-manual/4-integration/0-browser-extension.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Browser Extension"
+description: "SeaMeet Browser Extension"
+date: 2023-11-22T08:48:57+00:00
+lastmod: 2023-11-22T08:48:57+00:00
+draft: false
+images: []
+aliases:
+   - /en/seameet/4-integration/0-browser-extension/
+weight: 41
+toc: true
+---
+
+<br/>
+
+<div class="row justify-content-center">
+    <div class="col-lg-9 col-xl-8 text-center">
+        <p class="lead"></p>
+        <a class="btn btn-primary btn-lg px-4 mb-2" href="https://chrome.google.com/webstore/detail/seameet-ai-meeting-minute/gkkhkniggakfgioeeclbllpihmipkcmn" role="button">🚀 Install SeaMeet Extension</a>
+    </div>
+</div>
+
+<br/>
+
+SeaMeet is an easy-to-use meeting AI copilot that helps you transcribe, record, and analyze your meetings in REAL TIME.
+
+SeaMeet offers support to many different browsers, including Arc, Brave, and Edge. We recommend taking the first step by installing our browser extension, so you will always have SeaMeet on your side in meetings. Simply click the button above to install the SeaMeet extension.
```

Comment:
I got so frustrated with this...  Where's Chrome??? You shouldn't omit the most important browser we support please...